js/Domupdates.js

```diff
@@ -100,6 +101,25 @@ let domUpdates = {
     return result[0].toUpperCase() + result.substring(1);
   },
 
+  showPlayersTurn() {
```

Comment:
Nice I like the color sequence. Feels like a stop light, you even know when you on deck and next.